app/games/page.tsx

```
@@ -0,0 +1,1281 @@
+
+  const handleDeletePlayer = async (playerId: string) => {
+    try {
+      await removePlayerFromTeam(playerId);
+    } catch (error) {
+      console.error('Failed to delete player:', error);
+    }
+  };
+
+  const handleLoadSampleData = () => {
+    // No sample data available
+    if (false) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)